### DISCUSSION TITLE
Container Inspect HIGH security violation

### DISCUSSION THREAD

**Discussion by visingh-BRONGA** [kedacore/keda#2891] [Q&A / Need Help] [answered]
Hey - 

I plan to use KEDA in production on AKS (Azure Kubernetes Service) with Azure Service Bus (Scalar). The Container Inspect says security violation - "Ensure Containers are not permitted to run as root user.”  The CISO is not allowing to go production. 

Any suggested remedy/solution from the community.

**Comment by tomkerkhove** [comment, marked as answer]
Can you elaborate on how you did the installation?

**Reply by visingh-BRONGA** [reply]
Thanks for the response. These are the installation steps:

**Add helm repo**
$ helm repo add kedacore https://kedacore.github.io/charts
**Update repo**
$ helm repo update
**Install KEDA helm chart**
$ kubectl create namespace keda
$ helm install keda kedacore/keda --namespace keda
**Install parameters**
https://github.com/kedacore/charts/blob/main/keda/values.yaml

Any pointer to resolve this will be very much appreciated !

**Reply by tomkerkhove** [reply]
Tracking in https://github.com/kedacore/charts/issues/262
